db/migrate/20210801110000_create_students.rb

```diff
@@ -0,0 +1,13 @@
+class CreateStudents < ActiveRecord::Migration
+  def change
+    create_table :students do |t|
+      t.string :title, null:true, limit: 10
+      t.string :first_name, null:true, limit: 50
+      t.string :middle_name, null:true, limit: 50
```

Comment:
Small hint, `null: true` is the default and we only need to specify this parameter (as `null: false`) when the column is not nullable (ie. NOT NULL)

Reply:
Thanks for that Jordan...